src/semgrep_agent/semgrep.py

```diff
@@ -435,19 +435,27 @@ class SemgrepOutput:
 
 
 def invoke_semgrep(
-    semgrep_args: List[str], targets: List[str], *, timeout: Optional[int]
+    semgrep_args: List[str],
+    targets: List[str],
+    *,
+    timeout: Optional[int],
+    baseline: bool = False,
 ) -> Tuple[int, SemgrepOutput]:
     """
     Call semgrep passing in semgrep_args + targets as the arguments
     Also, save semgrep output as a list of json blobs in SEMGREP_SAVE_FILE
-    to help debugging
+    to help debugging. Baseline scan output will be saved separately with
+    the "_baseline" suffix.
 
     Returns json output of semgrep as dict object
     """
     max_exit_code = 0
     output = SemgrepOutput([], [], SemgrepTiming([], []))
 
-    semgrep_save_file = open(SEMGREP_SAVE_FILE, "w+")
+    semgrep_save_file_path = (
+        SEMGREP_SAVE_FILE + "_baseline" if baseline else SEMGREP_SAVE_FILE
```

Comment:
Let's still remove the baseline file before each run so that people can't get confused by it.

Otherwise, looks good to me!

Reply:
Addressed in https://github.com/returntocorp/semgrep-action/pull/472/commits/d8854e2370b361b166246932116572c4dc467ed4